terraform/aws/main.tf

```diff
@@ -61,6 +61,13 @@ resource "aws_security_group" "elb" {
     cidr_blocks = ["0.0.0.0/0"]
   }
 
+  ingress {
```

Comment:
Not sure why this is needed (the ELB should be forwarding from ELB port 80 to Insight host port 3001, so Insight nodes should not need to be accessible from the internet, just within the VPC subnet).

Reply:
80 port is already busy by mutlifaucet :( I've added a task to TODO to use subdomains or paths instead of different ports.

Reply:
Ok. This usually isn't considered best practice and I think better to let the load balancer handle it. For now, I don't want to hold up progress. But moving forward can use Application ELB and use paths to specify the backend target (e.g. `/insight` could go to Insight and `/faucet` could go to PHP faucet).